TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -233,6 +233,11 @@ void Moveable::Register(sol::table & parent)
 // shiva:SetObjectID(TEN.Objects.ObjID.BIGMEDI_ITEM)
 	ScriptReserved_SetObjectID, &Moveable::SetObjectID,
 
+/// Get the camera's index
+// @function Camera:GetIndex
+// @treturn short a copy of the camera index position in the level
+	ScriptReserved_GetIndex, &Moveable::GetIndex,
```

Comment:
I let them open for debugging uses, I had to use it internally to can get the object and camera in the level. And as I was using for check in gameplay, I though others could need it too.

But yes, they are optional for the lua Script (I can remove them from there).

However, I still need the functions internally to can get those indices. 
- Moveables have that index in its class, but is a protected var that I can't get it from Camera.
- And Camera hadn't got indices, so when I was looking for 1 camera in particular from the list, I couldn't, that's why I put an index to the camera item and a function to get it.

Reply:
Having the function internally (but not exposed to Lua) is fine I think :)

Reply:
Done it.

In this branch, I created a new var, Index for LevelCameraInfo objects, (it gets initialized on the Level.cpp load and is used in the lua CameraObject.cpp PlayCamera() function. 

From there, I made a function GetIndex() to can provide that value to external classes, however, at the end this was not used, so I deleted completely.

The other GetIndex in Moveable class, is still there, because is used by CameraObject.cpp to can get the protected index var. But now GetIndex function is internal and can't be called from Lua scripts.